test/api/v3/integration/tasks/groups/POST-group_tasks_id_score_direction.test.js

```diff
@@ -48,8 +53,8 @@ describe('POST /tasks/:id/score/:direction', () => {
       user: member.auth.local.username,
       taskName: updatedTask.text,
     }));
+    expect(user.notifications[0].data.groupId).to.equal(guild._id);
```

Comment:
These are all related to the expectations of what happens when the use case of seeking approval to score a task that needs approval. Are you wanting me to separate these expectations to a separate test more like a unit test?